### PR TITLE
Show avatar

### PR DESCRIPTION
**Pull request by susiecoleman**
Include avatar image on edit user page
Include the url for the avatar and public profile page as part of the Conf.scala file

### CODE REVIEW

File: app/config/Conf.scala [hunk 1]
```diff
@@ -8,6 +8,8 @@ trait Config {
   val baseUrl: String
   val baseRootUrl: String
   val errorEmail: String
+  val publicProfileUrl: String
```
Comment (nlindblad) [inline app/config/Conf.scala]: We might want to call these `base` as well? For clarity.

File: app/config/Conf.scala [hunk 2]
```diff
@@ -19,4 +21,10 @@ class MainConfig extends Config {
 
   val errorEmail = current.configuration.getString("identity-admin.email.error")
     .getOrElse(throw new IllegalStateException("Missing configuration: identity-admin.email.error"))
+
+  val publicProfileUrl = current.configuration.getString("identity-admin.editUser.baseProfileUrl")
```
Comment (nlindblad) [inline app/config/Conf.scala]: Same here.

File: app/views/editUser.scala.html [hunk 3]
```diff
@@ -28,10 +35,12 @@
                 }
 
                 <div class="row uneditable-fields">
-                    <div class="col-sm-2">
-                        <img src="@routes.Assets.at("images/darkUserIcon.svg")" />
+                    <div class="col-sm-1">
+                        <div class="avatar--margin--margin-top">
+                            <img class="avatar__circular__fixed-size" src="@avatarUrl@userId">
```
Comment (nlindblad) [inline app/views/editUser.scala.html]: Might be better to do the string concatenation before the view. At the moment it means the view is still performing some business logic. It should just be given the full public profile URL and avatar image URL for the user passed into it.

File: app/config/Conf.scala (hunk 1 above)
Comment (susiecoleman) [inline app/config/Conf.scala]: fixed

File: app/config/Conf.scala (hunk 2 above)
Comment (susiecoleman) [inline app/config/Conf.scala]: fixed

File: app/views/editUser.scala.html (hunk 3 above)
Comment (susiecoleman) [inline app/views/editUser.scala.html]: fixed

**Comment by nlindblad** [comment]
:+1:

**Pull request merged**